RELEASE-NOTES.rst

```diff
@@ -18,6 +18,243 @@
 Release notes
 =============
 
+v0.6.0
+------
+
+User interface
+--------------
+
+-  Adds a “renew” button in the patron profile, to allow the patron to
+   renew the borrowed item, if possible (no request on it).
+-  Re-enables the autocomplete in the search input.
+-  Separation of the user interface into a public interface and a
+   professional interface (ie for librarians).
+-  Removes all professional actions from the public interface, as they
+   are moved to the professional interface.
+-  Move all professional functionalities to a dedicated angular
+   application.
+-  Adds a link to switch to the professional interface (only available
+   to logged in librarians).
+-  Adds a link to switch from the professional interface to the public
+   interface.
+-  Filters persons by organisation views.
+-  Changes the angular library to generate forms (form
+   `angular6-json-schema-editor <https://github.com/hamzahamidi/ajsf>`__
+   to `ngx-formly <https://github.com/ngx-formly/ngx-formly>`__), in
+   order to come up with the need of a complex cataloging editor.
+-  Displays custom logos and header color for each organisation and the
+   professional interface.
+-  Adds an “history” tab in the patron profile to display transactions
+   history of the last 6 months.
+-  Adds a button in the requests tab of the patron profile to allow
+   patrons to cancel their own requests.
+-  Updates the frontpage for the pilot libraries instance, with relevant
+   information.
+
+Search and indexing
+-------------------
+
+-  Improves ebooks bulk indexing (``invenio utils reindex``,
+   ``invenio utils runindex``).
+-  Improves persons indexing during document indexing and document
+   creation.
+-  Fixes missing mappings in JSON schemas.
+-  Adds default sorting for each resource.
+
+Circulation
+-----------
+
+-  Adds a method to get the timezone of a library to improves
+   computation of a checkout due date.
+-  Links loans and fees.
```

Comment:
The links between the resources fee and loan are now made via the resource notification